[
    {
        "log": "\"],\"command\":[\"/bin/bash\",\"-c\"],\"image\":\"haproxy:latest\",\"name\":\"haproxy-container\",\"ports\":{\"containerPort\":80}}],\"dnsConfig\":{\"nameservers\":[\"10.99.30.112\"]},\"dnsPolicy\":\"None\",\"enableServiceLinks\":true,\"nodeName\":\"kube-172\",\"preemptionPolicy\":\"PreemptLowerPriority\",\"priority\":0,\"restartPolicy\":\"Always\",\"schedulerName\":\"default-scheduler\",\"securityContext\":{},\"serviceAccount\":\"default\",\"serviceAccountName\":\"default\",\"terminationGracePeriodSeconds\":30,\"tolerations\":[{\"key\":\"node.kubernetes.io/not-ready\",\"operator\":\"Exists\",\"effect\":\"NoExecute\",\"tolerationSeconds\":300},{\"key\":\"node.kubernetes.io/unreachable\",\"operator\":\"Exists\",\"effect\":\"NoExecute\",\"tolerationSeconds\":300}],\"volumes\":[{\"name\":\"kube-api-access-zpr5t\",\"projected\":{\"sources\":[{\"serviceAccountToken\":{\"expirationSeconds\":3607,\"path\":\"token\"}},{\"configMap\":{\"name\":\"kube-root-ca.crt\",\"items\":[{\"key\":\"ca.crt\",\"path\":\"ca.crt\"}]}},{\"downwardAPI\":{\"items\":[{\"path\":\"namespace\",\"fieldRef\":{\"apiVersion\":\"v1\",\"fieldPath\":\"metadata.namespace\"}}]}}],\"defaultMode\":420}}]},\"status\":{\"phase\":\"Running\",\"conditions\":[{\"type\":\"PodReadyToStartContainers\",\"status\":\"True\",\"lastProbeTime\":null,\"lastTransitionTime\":\"2025-05-29T09:42:14Z\"},{\"type\":\"Initialized\",\"status\":\"True\",\"lastProbeTime\":null,\"lastTransitionTime\":\"2025-05-29T09:42:11Z\"},{\"type\":\"Ready\",\"status\":\"True\",\"lastProbeTime\":null,\"lastTransitionTime\":\"2025-05-29T09:42:14Z\"},{\"type\":\"ContainersReady\",\"status\":\"True\",\"lastProbeTime\":null,\"lastTransitionTime\":\"2025-05-29T09:42:14Z\"},{\"type\":\"PodScheduled\",\"status\":\"True\",\"lastProbeTime\":null,\"lastTransitionTime\":\"2025-05-29T09:42:11Z\"}],\"hostIP\":\"141.223.181.172\",\"hostIPs\":[{\"ip\":\"141.223.181.172\"}],\"podIP\":\"10.244.12.27\",\"podIPs\":[{\"ip\":\"10.244.12.27\"}],\"startTime\":\"2025-05-29T09:42:11Z\",\"containerStatuses\":[{\"name\":\"haproxy-container\",\"state\":{\"running\":{\"startedAt\":\"2025-05-29T09:42:13Z\"}},\"lastState\":{},\"ready\":true,\"restartCount\":0,\"image\":\"docker.io/dokken/ubuntu-20.04:latest\",\"imageID\":\"docker.io/dokken/ubuntu-20.04@sha256:61c700117df72aeeb84838caf8f408be3bdd6e776f1e5696d1fdfeea0c3ca4e0\",\"containerID\":\"containerd://f9e3aeb1ad3129381436d1e094329ae455d130e3387e70e428ce84c2820fa9eb\",\"started\":true}],\"qosClass\":\"Burstable\"}}\": json: cannot unmarshal object into Go struct field Container.spec.containers.ports of type []v1.ContainerPort\",\"reason\":\"Invalid\",\"details\":{\"causes\":[{\"reason\":\"FieldValueInvalid\",\"message\":\"Invalid value: \"{\"apiVersion\":\"v1\",\"kind\":\"Pod\",\"metadata\":{\"creationTimestamp\":\"2025-05-29T09:42:11Z\",\"deletionGracePeriodSeconds\":30,\"deletionTimestamp\":\"2025-05-29T09:43:48Z\",\"labels\":{\"app\":\"haproxy\"},\"managedFields\":[{\"manager\":\"OpenAPI-Generator\",\"operation\":\"Update\",\"apiVersion\":\"v1\",\"time\":\"2025-05-29T09:42:11Z\",\"fieldsType\":\"FieldsV1\",\"fieldsV1\":{\"f:metadata\":{\"f:labels\":{\".\":{},\"f:app\":{}}},\"f:spec\":{\"f:containers\":{\"k:{\"name\":\"haproxy-container\"}\":{\".\":{},\"f:args\":{},\"f:command\":{},\"f:image\":{},\"f:imagePullPolicy\":{},\"f:name\":{},\"f:resources\":{\".\":{},\"f:limits\":{\".\":{},\"f:cpu\":{},\"f:memory\":{}},\"f:requests\":{\".\":{},\"f:cpu\":{},\"f:memory\":{}}},\"f:securityContext\":{\".\":{},\"f:privileged\":{}},\"f:terminationMessagePath\":{},\"f:terminationMessagePolicy\":{}}},\"f:dnsConfig\":{\".\":{},\"f:nameservers\":{}},\"f:dnsPolicy\":{},\"f:enableServiceLinks\":{},\"f:restartPolicy\":{},\"f:schedulerName\":{},\"f:securityContext\":{},\"f:terminationGracePeriodSeconds\":{}}}},{\"manager\":\"kubelet\",\"operation\":\"Update\",\"apiVersion\":\"v1\",\"time\":\"2025-05-29T09:42:14Z\",\"fieldsType\":\"FieldsV1\",\"fieldsV1\":{\"f:status\":{\"f:conditions\":{\"k:{\"type\":\"ContainersReady\"}\":{\".\":{},\"f:lastProbeTime\":{},\"f:lastTransitionTime\":{},\"f:status\":{},\"f:type\":{}},\"k:{\"type\":\"Initialized\"}\":{\".\":{},\"f:lastProbeTime\":{},\"f:lastTransitionTime\":{},\"f:status\":{},\"f:type\":{}},\"k:{\"type\":\"PodReadyToStartContainers\"}\":{\".\":{},\"f:lastProbeTime\":{},\"f:lastTransitionTime\":{},\"f:status\":{},\"f:type\":{}},\"k:{\"type\":\"Ready\"}\":{\".\":{},\"f:lastProbeTime\":{},\"f:lastTransitionTime\":{},\"f:status\":{},\"f:type\":{}}},\"f:containerStatuses\":{},\"f:hostIP\":{},\"f:hostIPs\":{},\"f:phase\":{},\"f:podIP\":{},\"f:podIPs\":{\".\":{},\"k:{\"ip\":\"10.244.12.27\"}\":{\".\":{},\"f:ip\":{}}},\"f:startTime\":{}}},\"subresource\":\"status\"}],\"name\":\"haproxy-pod\",\"namespace\":\"llm-config\",\"resourceVersion\":\"11372666\",\"uid\":\"d7506c06-cef6-4625-9396-61a8bd69db62\"},\"spec\":{\"containers\":[{\"args\":[\"apt update && apt install -y software-properties-common",
        "title": "Service in version \"v1\" cannot be handled as a Service: json: cannot unmarshal object into Go struct field ServiceSpec.spec.ports of type []v1.Service"
    },
    {
        "log": "\"],\"command\":[\"/bin/bash\",\"-c\"],\"image\":\"haproxy:latest\",\"name\":\"haproxy-container\",\"ports\":{\"containerPort\":80}}],\"dnsConfig\":{\"nameservers\":[\"10.99.30.112\"]},\"dnsPolicy\":\"None\",\"enableServiceLinks\":true,\"nodeName\":\"kube-172\",\"preemptionPolicy\":\"PreemptLowerPriority\",\"priority\":0,\"restartPolicy\":\"Always\",\"schedulerName\":\"default-scheduler\",\"securityContext\":{},\"serviceAccount\":\"default\",\"serviceAccountName\":\"default\",\"terminationGracePeriodSeconds\":30,\"tolerations\":[{\"key\":\"node.kubernetes.io/not-ready\",\"operator\":\"Exists\",\"effect\":\"NoExecute\",\"tolerationSeconds\":300},{\"key\":\"node.kubernetes.io/unreachable\",\"operator\":\"Exists\",\"effect\":\"NoExecute\",\"tolerationSeconds\":300}],\"volumes\":[{\"name\":\"kube-api-access-zpr5t\",\"projected\":{\"sources\":[{\"serviceAccountToken\":{\"expirationSeconds\":3607,\"path\":\"token\"}},{\"configMap\":{\"name\":\"kube-root-ca.crt\",\"items\":[{\"key\":\"ca.crt\",\"path\":\"ca.crt\"}]}},{\"downwardAPI\":{\"items\":[{\"path\":\"namespace\",\"fieldRef\":{\"apiVersion\":\"v1\",\"fieldPath\":\"metadata.namespace\"}}]}}],\"defaultMode\":420}}]},\"status\":{\"phase\":\"Running\",\"conditions\":[{\"type\":\"PodReadyToStartContainers\",\"status\":\"True\",\"lastProbeTime\":null,\"lastTransitionTime\":\"2025-05-29T09:42:14Z\"},{\"type\":\"Initialized\",\"status\":\"True\",\"lastProbeTime\":null,\"lastTransitionTime\":\"2025-05-29T09:42:11Z\"},{\"type\":\"Ready\",\"status\":\"True\",\"lastProbeTime\":null,\"lastTransitionTime\":\"2025-05-29T09:42:14Z\"},{\"type\":\"ContainersReady\",\"status\":\"True\",\"lastProbeTime\":null,\"lastTransitionTime\":\"2025-05-29T09:42:14Z\"},{\"type\":\"PodScheduled\",\"status\":\"True\",\"lastProbeTime\":null,\"lastTransitionTime\":\"2025-05-29T09:42:11Z\"}],\"hostIP\":\"141.223.181.172\",\"hostIPs\":[{\"ip\":\"141.223.181.172\"}],\"podIP\":\"10.244.12.27\",\"podIPs\":[{\"ip\":\"10.244.12.27\"}],\"startTime\":\"2025-05-29T09:42:11Z\",\"containerStatuses\":[{\"name\":\"haproxy-container\",\"state\":{\"running\":{\"startedAt\":\"2025-05-29T09:42:13Z\"}},\"lastState\":{},\"ready\":true,\"restartCount\":0,\"image\":\"docker.io/dokken/ubuntu-20.04:latest\",\"imageID\":\"docker.io/dokken/ubuntu-20.04@sha256:61c700117df72aeeb84838caf8f408be3bdd6e776f1e5696d1fdfeea0c3ca4e0\",\"containerID\":\"containerd://f9e3aeb1ad3129381436d1e094329ae455d130e3387e70e428ce84c2820fa9eb\",\"started\":true}],\"qosClass\":\"Burstable\"}}\": json: cannot unmarshal object into Go struct field Container.spec.containers.ports of type []v1.ContainerPort\",\"field\":\"patch\"}]},\"code\":422}",
        "title": "Service in version \"v1\" cannot be handled as a Service: json: cannot unmarshal object into Go struct field ServiceSpec.spec.ports of type []v1.Service"
    },
    {
        "log": "\"],\"command\":[\"/bin/bash\",\"-c\"],\"image\":\"alexellis/nmap\",\"name\":\"firewall\"}],\"dnsConfig\":{\"nameservers\":[\"10.99.30.112\"]},\"dnsPolicy\":\"None\",\"enableServiceLinks\":true,\"nodeName\":\"kube-172\",\"preemptionPolicy\":\"PreemptLowerPriority\",\"priority\":0,\"restartPolicy\":\"Always\",\"schedulerName\":\"default-scheduler\",\"securityContext\":{},\"serviceAccount\":\"default\",\"serviceAccountName\":\"default\",\"terminationGracePeriodSeconds\":30,\"tolerations\":[{\"key\":\"node.kubernetes.io/not-ready\",\"operator\":\"Exists\",\"effect\":\"NoExecute\",\"tolerationSeconds\":300},{\"key\":\"node.kubernetes.io/unreachable\",\"operator\":\"Exists\",\"effect\":\"NoExecute\",\"tolerationSeconds\":300}],\"volumes\":[{\"name\":\"kube-api-access-ngnpn\",\"projected\":{\"sources\":[{\"serviceAccountToken\":{\"expirationSeconds\":3607,\"path\":\"token\"}},{\"configMap\":{\"name\":\"kube-root-ca.crt\",\"items\":[{\"key\":\"ca.crt\",\"path\":\"ca.crt\"}]}},{\"downwardAPI\":{\"items\":[{\"path\":\"namespace\",\"fieldRef\":{\"apiVersion\":\"v1\",\"fieldPath\":\"metadata.namespace\"}}]}}],\"defaultMode\":420}}]},\"status\":{\"phase\":\"Running\",\"conditions\":[{\"type\":\"PodReadyToStartContainers\",\"status\":\"True\",\"lastProbeTime\":null,\"lastTransitionTime\":\"2025-05-29T10:59:36Z\"},{\"type\":\"Initialized\",\"status\":\"True\",\"lastProbeTime\":null,\"lastTransitionTime\":\"2025-05-29T10:59:33Z\"},{\"type\":\"Ready\",\"status\":\"True\",\"lastProbeTime\":null,\"lastTransitionTime\":\"2025-05-29T10:59:36Z\"},{\"type\":\"ContainersReady\",\"status\":\"True\",\"lastProbeTime\":null,\"lastTransitionTime\":\"2025-05-29T10:59:36Z\"},{\"type\":\"PodScheduled\",\"status\":\"True\",\"lastProbeTime\":null,\"lastTransitionTime\":\"2025-05-29T10:59:33Z\"}],\"hostIP\":\"141.223.181.172\",\"hostIPs\":[{\"ip\":\"141.223.181.172\"}],\"podIP\":\"10.244.12.63\",\"podIPs\":[{\"ip\":\"10.244.12.63\"}],\"startTime\":\"2025-05-29T10:59:33Z\",\"containerStatuses\":[{\"name\":\"firewall-container\",\"state\":{\"running\":{\"startedAt\":\"2025-05-29T10:59:35Z\"}},\"lastState\":{},\"ready\":true,\"restartCount\":0,\"image\":\"docker.io/dokken/ubuntu-20.04:latest\",\"imageID\":\"docker.io/dokken/ubuntu-20.04@sha256:61c700117df72aeeb84838caf8f408be3bdd6e776f1e5696d1fdfeea0c3ca4e0\",\"containerID\":\"containerd://ce4c18a8d91b6b856f2781c69c8ee502685bc78aecf12c6a4f01d7e78526e0e4\",\"started\":true}],\"qosClass\":\"BestEffort\"}}\": json: cannot unmarshal object into Go struct field Container.spec.containers.ports of type []v1.ContainerPort\",\"reason\":\"Invalid\",\"details\":{\"causes\":[{\"reason\":\"FieldValueInvalid\",\"message\":\"Invalid value: \"{\"apiVersion\":\"v1\",\"kind\":\"Pod\",\"metadata\":{\"creationTimestamp\":\"2025-05-29T10:59:33Z\",\"deletionGracePeriodSeconds\":30,\"deletionTimestamp\":\"2025-05-29T11:02:42Z\",\"managedFields\":[{\"manager\":\"OpenAPI-Generator\",\"operation\":\"Update\",\"apiVersion\":\"v1\",\"time\":\"2025-05-29T10:59:33Z\",\"fieldsType\":\"FieldsV1\",\"fieldsV1\":{\"f:spec\":{\"f:containers\":{\"k:{\"name\":\"firewall-container\"}\":{\".\":{},\"f:args\":{},\"f:command\":{},\"f:image\":{},\"f:imagePullPolicy\":{},\"f:name\":{},\"f:resources\":{},\"f:securityContext\":{\".\":{},\"f:privileged\":{}},\"f:terminationMessagePath\":{},\"f:terminationMessagePolicy\":{}}},\"f:dnsConfig\":{\".\":{},\"f:nameservers\":{}},\"f:dnsPolicy\":{},\"f:enableServiceLinks\":{},\"f:restartPolicy\":{},\"f:schedulerName\":{},\"f:securityContext\":{},\"f:terminationGracePeriodSeconds\":{}}}},{\"manager\":\"kubelet\",\"operation\":\"Update\",\"apiVersion\":\"v1\",\"time\":\"2025-05-29T10:59:36Z\",\"fieldsType\":\"FieldsV1\",\"fieldsV1\":{\"f:status\":{\"f:conditions\":{\"k:{\"type\":\"ContainersReady\"}\":{\".\":{},\"f:lastProbeTime\":{},\"f:lastTransitionTime\":{},\"f:status\":{},\"f:type\":{}},\"k:{\"type\":\"Initialized\"}\":{\".\":{},\"f:lastProbeTime\":{},\"f:lastTransitionTime\":{},\"f:status\":{},\"f:type\":{}},\"k:{\"type\":\"PodReadyToStartContainers\"}\":{\".\":{},\"f:lastProbeTime\":{},\"f:lastTransitionTime\":{},\"f:status\":{},\"f:type\":{}},\"k:{\"type\":\"Ready\"}\":{\".\":{},\"f:lastProbeTime\":{},\"f:lastTransitionTime\":{},\"f:status\":{},\"f:type\":{}}},\"f:containerStatuses\":{},\"f:hostIP\":{},\"f:hostIPs\":{},\"f:phase\":{},\"f:podIP\":{},\"f:podIPs\":{\".\":{},\"k:{\"ip\":\"10.244.12.63\"}\":{\".\":{},\"f:ip\":{}}},\"f:startTime\":{}}},\"subresource\":\"status\"}],\"name\":\"firewall-pod\",\"namespace\":\"llm-config\",\"resourceVersion\":\"11386050\",\"uid\":\"57ad2958-d31e-41aa-aae1-b1f313c82df5\"},\"spec\":{\"containers\":[{\"args\":[\"apt update && apt install -y software-properties-common",
        "title": "Service in version \"v1\" cannot be handled as a Service: json: cannot unmarshal object into Go struct field ServiceSpec.spec.ports of type []v1.Service"
    },
    {
        "log": "\"],\"command\":[\"/bin/bash\",\"-c\"],\"image\":\"alexellis/nmap\",\"name\":\"firewall\"}],\"dnsConfig\":{\"nameservers\":[\"10.99.30.112\"]},\"dnsPolicy\":\"None\",\"enableServiceLinks\":true,\"nodeName\":\"kube-172\",\"preemptionPolicy\":\"PreemptLowerPriority\",\"priority\":0,\"restartPolicy\":\"Always\",\"schedulerName\":\"default-scheduler\",\"securityContext\":{},\"serviceAccount\":\"default\",\"serviceAccountName\":\"default\",\"terminationGracePeriodSeconds\":30,\"tolerations\":[{\"key\":\"node.kubernetes.io/not-ready\",\"operator\":\"Exists\",\"effect\":\"NoExecute\",\"tolerationSeconds\":300},{\"key\":\"node.kubernetes.io/unreachable\",\"operator\":\"Exists\",\"effect\":\"NoExecute\",\"tolerationSeconds\":300}],\"volumes\":[{\"name\":\"kube-api-access-ngnpn\",\"projected\":{\"sources\":[{\"serviceAccountToken\":{\"expirationSeconds\":3607,\"path\":\"token\"}},{\"configMap\":{\"name\":\"kube-root-ca.crt\",\"items\":[{\"key\":\"ca.crt\",\"path\":\"ca.crt\"}]}},{\"downwardAPI\":{\"items\":[{\"path\":\"namespace\",\"fieldRef\":{\"apiVersion\":\"v1\",\"fieldPath\":\"metadata.namespace\"}}]}}],\"defaultMode\":420}}]},\"status\":{\"phase\":\"Running\",\"conditions\":[{\"type\":\"PodReadyToStartContainers\",\"status\":\"True\",\"lastProbeTime\":null,\"lastTransitionTime\":\"2025-05-29T10:59:36Z\"},{\"type\":\"Initialized\",\"status\":\"True\",\"lastProbeTime\":null,\"lastTransitionTime\":\"2025-05-29T10:59:33Z\"},{\"type\":\"Ready\",\"status\":\"True\",\"lastProbeTime\":null,\"lastTransitionTime\":\"2025-05-29T10:59:36Z\"},{\"type\":\"ContainersReady\",\"status\":\"True\",\"lastProbeTime\":null,\"lastTransitionTime\":\"2025-05-29T10:59:36Z\"},{\"type\":\"PodScheduled\",\"status\":\"True\",\"lastProbeTime\":null,\"lastTransitionTime\":\"2025-05-29T10:59:33Z\"}],\"hostIP\":\"141.223.181.172\",\"hostIPs\":[{\"ip\":\"141.223.181.172\"}],\"podIP\":\"10.244.12.63\",\"podIPs\":[{\"ip\":\"10.244.12.63\"}],\"startTime\":\"2025-05-29T10:59:33Z\",\"containerStatuses\":[{\"name\":\"firewall-container\",\"state\":{\"running\":{\"startedAt\":\"2025-05-29T10:59:35Z\"}},\"lastState\":{},\"ready\":true,\"restartCount\":0,\"image\":\"docker.io/dokken/ubuntu-20.04:latest\",\"imageID\":\"docker.io/dokken/ubuntu-20.04@sha256:61c700117df72aeeb84838caf8f408be3bdd6e776f1e5696d1fdfeea0c3ca4e0\",\"containerID\":\"containerd://ce4c18a8d91b6b856f2781c69c8ee502685bc78aecf12c6a4f01d7e78526e0e4\",\"started\":true}],\"qosClass\":\"BestEffort\"}}\": json: cannot unmarshal object into Go struct field Container.spec.containers.ports of type []v1.ContainerPort\",\"field\":\"patch\"}]},\"code\":422}",
        "title": "Service in version \"v1\" cannot be handled as a Service: json: cannot unmarshal object into Go struct field ServiceSpec.spec.ports of type []v1.Service"
    },
    {
        "log": "\",\"reason\":\"Invalid\",\"details\":{\"name\":\"ndpi-pod\",\"kind\":\"Pod\",\"causes\":[{\"reason\":\"FieldValueForbidden\",\"message\":\"Forbidden: pod updates may not change fields other than `spec.containers[*].image`,`spec.initContainers[*].image`,`spec.activeDeadlineSeconds`,`spec.tolerations` (only additions to existing tolerations),`spec.terminationGracePeriodSeconds` (allow it to be set to 1 if it was previously negative)",
        "title": "Invalid spec when I run pod.yaml"
    },
    {
        "log": "\",\"reason\":\"Invalid\",\"details\":{\"name\":\"suricata-pod\",\"kind\":\"Pod\",\"causes\":[{\"reason\":\"FieldValueForbidden\",\"message\":\"Forbidden: pod updates may not change fields other than `spec.containers[*].image`,`spec.initContainers[*].image`,`spec.activeDeadlineSeconds`,`spec.tolerations` (only additions to existing tolerations),`spec.terminationGracePeriodSeconds` (allow it to be set to 1 if it was previously negative)",
        "title": "Invalid spec when I run pod.yaml"
    },
    {
        "log": "\",\"reason\":\"Invalid\",\"details\":{\"name\":\"ntopng-pod\",\"kind\":\"Pod\",\"causes\":[{\"reason\":\"FieldValueForbidden\",\"message\":\"Forbidden: pod updates may not change fields other than `spec.containers[*].image`,`spec.initContainers[*].image`,`spec.activeDeadlineSeconds`,`spec.tolerations` (only additions to existing tolerations),`spec.terminationGracePeriodSeconds` (allow it to be set to 1 if it was previously negative)",
        "title": "Invalid spec when I run pod.yaml"
    },
    {
        "log": "]\",\"reason\":\"Invalid\",\"details\":{\"name\":\"firewall-pod\",\"kind\":\"Pod\",\"causes\":[{\"reason\":\"FieldValueInvalid\",\"message\":\"Invalid value: \"Burstable\": Pod QoS is immutable\",\"field\":\"metadata\"},{\"reason\":\"FieldValueForbidden\",\"message\":\"Forbidden: pod updates may not change fields other than `spec.containers[*].image`,`spec.initContainers[*].image`,`spec.activeDeadlineSeconds`,`spec.tolerations` (only additions to existing tolerations),`spec.terminationGracePeriodSeconds` (allow it to be set to 1 if it was previously negative)",
        "title": "Invalid spec when I run pod.yaml"
    },
    {
        "log": "Err:1 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 haproxy amd64 2.0.33-0ubuntu0.1",
        "title": "apt-get update fails to fetch files, \u201cTemporary failure resolving \u2026\u201d error"
    },
    {
        "log": "Temporary failure resolving 'archive.ubuntu.com'",
        "title": "apt-get update fails to fetch files, \u201cTemporary failure resolving \u2026\u201d error"
    },
    {
        "log": "E: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/h/haproxy/haproxy_2.0.33-0ubuntu0.1_amd64.deb Temporary failure resolving 'archive.ubuntu.com'",
        "title": "apt-get update fails to fetch files, \u201cTemporary failure resolving \u2026\u201d error"
    },
    {
        "log": "E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?",
        "title": "apt-get update fails to fetch files, \u201cTemporary failure resolving \u2026\u201d error"
    },
    {
        "log": "/bin/bash: line 27: haproxy: command not found",
        "title": "haproxy: command not found"
    },
    {
        "log": "Container killed, usually due to out of memory (OOM) or manual termination",
        "title": "container restart reason OOMKilled with exit code 1"
    },
    {
        "log": "Err:12 http://archive.ubuntu.com/ubuntu focal/main amd64 manpages all 5.05-1",
        "title": "apt-get update fails to fetch files, \u201cTemporary failure resolving \u2026\u201d error"
    },
    {
        "log": "Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.39), connection timed out Could not connect to archive.ubuntu.com:80 (185.125.190.81), connection timed out Could not connect to archive.ubuntu.com:80 (185.125.190.82), connection timed out Could not connect to archive.ubuntu.com:80 (185.125.190.36), connection timed out [IP: 185.125.190.39 80]",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Err:13 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 python-apt-common all 2.0.1ubuntu0.20.04.1",
        "title": "apt-get update fails to fetch files, \u201cTemporary failure resolving \u2026\u201d error"
    },
    {
        "log": "Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) [IP: 185.125.190.39 80]",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Err:14 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 python3-apt amd64 2.0.1ubuntu0.20.04.1",
        "title": "apt-get update fails to fetch files, \u201cTemporary failure resolving \u2026\u201d error"
    },
    {
        "log": "Err:15 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 python3-distro-info all 0.23ubuntu1.1",
        "title": "apt-get update fails to fetch files, \u201cTemporary failure resolving \u2026\u201d error"
    },
    {
        "log": "Err:24 http://archive.ubuntu.com/ubuntu focal/main amd64 libcrypt-dev amd64 1:4.4.10-10ubuntu4",
        "title": "apt-get update fails to fetch files, \u201cTemporary failure resolving \u2026\u201d error"
    },
    {
        "log": "Err:27 http://archive.ubuntu.com/ubuntu focal/main amd64 libisl22 amd64 0.22.1-1",
        "title": "apt-get update fails to fetch files, \u201cTemporary failure resolving \u2026\u201d error"
    },
    {
        "log": "Err:28 http://archive.ubuntu.com/ubuntu focal/main amd64 libmpfr6 amd64 4.0.2-1",
        "title": "apt-get update fails to fetch files, \u201cTemporary failure resolving \u2026\u201d error"
    },
    {
        "log": "Err:29 http://archive.ubuntu.com/ubuntu focal/main amd64 libmpc3 amd64 1.1.0-1",
        "title": "apt-get update fails to fetch files, \u201cTemporary failure resolving \u2026\u201d error"
    },
    {
        "log": "Err:31 http://archive.ubuntu.com/ubuntu focal/main amd64 cpp amd64 4:9.3.0-1ubuntu2",
        "title": "apt-get update fails to fetch files, \u201cTemporary failure resolving \u2026\u201d error"
    },
    {
        "log": "Err:43 http://archive.ubuntu.com/ubuntu focal/main amd64 gcc amd64 4:9.3.0-1ubuntu2",
        "title": "apt-get update fails to fetch files, \u201cTemporary failure resolving \u2026\u201d error"
    },
    {
        "log": "Err:46 http://archive.ubuntu.com/ubuntu focal/main amd64 g++ amd64 4:9.3.0-1ubuntu2",
        "title": "apt-get update fails to fetch files, \u201cTemporary failure resolving \u2026\u201d error"
    },
    {
        "log": "Err:49 http://archive.ubuntu.com/ubuntu focal/main amd64 patch amd64 2.7.6-6",
        "title": "apt-get update fails to fetch files, \u201cTemporary failure resolving \u2026\u201d error"
    },
    {
        "log": "Err:51 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 build-essential amd64 12.8ubuntu1.1",
        "title": "apt-get update fails to fetch files, \u201cTemporary failure resolving \u2026\u201d error"
    },
    {
        "log": "Err:52 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 cmake-data all 3.16.3-1ubuntu1.20.04.1",
        "title": "apt-get update fails to fetch files, \u201cTemporary failure resolving \u2026\u201d error"
    },
    {
        "log": "Err:54 http://archive.ubuntu.com/ubuntu focal/main amd64 libjsoncpp1 amd64 1.7.4-3.1ubuntu2",
        "title": "apt-get update fails to fetch files, \u201cTemporary failure resolving \u2026\u201d error"
    },
    {
        "log": "Err:55 http://archive.ubuntu.com/ubuntu focal/main amd64 librhash0 amd64 1.3.9-1",
        "title": "apt-get update fails to fetch files, \u201cTemporary failure resolving \u2026\u201d error"
    },
    {
        "log": "Err:56 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 cmake amd64 3.16.3-1ubuntu1.20.04.1",
        "title": "apt-get update fails to fetch files, \u201cTemporary failure resolving \u2026\u201d error"
    },
    {
        "log": "Err:57 http://archive.ubuntu.com/ubuntu focal/main amd64 libdconf1 amd64 0.36.0-1",
        "title": "apt-get update fails to fetch files, \u201cTemporary failure resolving \u2026\u201d error"
    },
    {
        "log": "Err:58 http://archive.ubuntu.com/ubuntu focal/main amd64 dconf-service amd64 0.36.0-1",
        "title": "apt-get update fails to fetch files, \u201cTemporary failure resolving \u2026\u201d error"
    },
    {
        "log": "Err:59 http://archive.ubuntu.com/ubuntu focal/main amd64 dconf-gsettings-backend amd64 0.36.0-1",
        "title": "apt-get update fails to fetch files, \u201cTemporary failure resolving \u2026\u201d error"
    },
    {
        "log": "Err:60 http://archive.ubuntu.com/ubuntu focal/main amd64 libfakeroot amd64 1.24-1",
        "title": "apt-get update fails to fetch files, \u201cTemporary failure resolving \u2026\u201d error"
    },
    {
        "log": "Err:61 http://archive.ubuntu.com/ubuntu focal/main amd64 fakeroot amd64 1.24-1",
        "title": "apt-get update fails to fetch files, \u201cTemporary failure resolving \u2026\u201d error"
    },
    {
        "log": "Err:65 http://archive.ubuntu.com/ubuntu focal/main amd64 liberror-perl all 0.17029-1",
        "title": "apt-get update fails to fetch files, \u201cTemporary failure resolving \u2026\u201d error"
    },
    {
        "log": "Err:71 http://archive.ubuntu.com/ubuntu focal/main amd64 gsettings-desktop-schemas all 3.36.0-1ubuntu1",
        "title": "apt-get update fails to fetch files, \u201cTemporary failure resolving \u2026\u201d error"
    },
    {
        "log": "Err:73 http://archive.ubuntu.com/ubuntu focal/main amd64 libalgorithm-diff-perl all 1.19.03-2",
        "title": "apt-get update fails to fetch files, \u201cTemporary failure resolving \u2026\u201d error"
    },
    {
        "log": "Err:74 http://archive.ubuntu.com/ubuntu focal/main amd64 libalgorithm-diff-xs-perl amd64 0.04-6",
        "title": "apt-get update fails to fetch files, \u201cTemporary failure resolving \u2026\u201d error"
    },
    {
        "log": "Err:75 http://archive.ubuntu.com/ubuntu focal/main amd64 libalgorithm-merge-perl all 0.08-3",
        "title": "apt-get update fails to fetch files, \u201cTemporary failure resolving \u2026\u201d error"
    },
    {
        "log": "Err:77 http://archive.ubuntu.com/ubuntu focal/main amd64 libstemmer0d amd64 0+svn585-2",
        "title": "apt-get update fails to fetch files, \u201cTemporary failure resolving \u2026\u201d error"
    },
    {
        "log": "Err:78 http://archive.ubuntu.com/ubuntu focal/main amd64 libappstream4 amd64 0.12.10-2",
        "title": "apt-get update fails to fetch files, \u201cTemporary failure resolving \u2026\u201d error"
    },
    {
        "log": "Err:79 http://archive.ubuntu.com/ubuntu focal/main amd64 libfile-fcntllock-perl amd64 0.22-3build4",
        "title": "apt-get update fails to fetch files, \u201cTemporary failure resolving \u2026\u201d error"
    },
    {
        "log": "Err:82 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 libpcap0.8-dev amd64 1.9.1-3ubuntu1.20.04.1",
        "title": "apt-get update fails to fetch files, \u201cTemporary failure resolving \u2026\u201d error"
    },
    {
        "log": "Err:83 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 libpcap-dev amd64 1.9.1-3ubuntu1.20.04.1",
        "title": "apt-get update fails to fetch files, \u201cTemporary failure resolving \u2026\u201d error"
    },
    {
        "log": "Err:85 http://archive.ubuntu.com/ubuntu focal/main amd64 manpages-dev all 5.05-1",
        "title": "apt-get update fails to fetch files, \u201cTemporary failure resolving \u2026\u201d error"
    },
    {
        "log": "Err:89 http://archive.ubuntu.com/ubuntu focal/main amd64 python3-certifi all 2019.11.28-1",
        "title": "apt-get update fails to fetch files, \u201cTemporary failure resolving \u2026\u201d error"
    },
    {
        "log": "Err:90 http://archive.ubuntu.com/ubuntu focal/main amd64 python3-chardet all 3.0.4-4build1",
        "title": "apt-get update fails to fetch files, \u201cTemporary failure resolving \u2026\u201d error"
    },
    {
        "log": "Err:94 http://archive.ubuntu.com/ubuntu focal/main amd64 python3-requests-unixsocket all 0.2.0-2",
        "title": "apt-get update fails to fetch files, \u201cTemporary failure resolving \u2026\u201d error"
    },
    {
        "log": "Err:95 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 python3-software-properties all 0.99.9.12",
        "title": "apt-get update fails to fetch files, \u201cTemporary failure resolving \u2026\u201d error"
    },
    {
        "log": "Err:96 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 software-properties-common all 0.99.9.12",
        "title": "apt-get update fails to fetch files, \u201cTemporary failure resolving \u2026\u201d error"
    },
    {
        "log": "Err:97 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 unattended-upgrades all 2.3ubuntu0.3",
        "title": "apt-get update fails to fetch files, \u201cTemporary failure resolving \u2026\u201d error"
    },
    {
        "log": "Make Error: The source directory \"/tmp/nDPI\" does not appear to contain CMakeLists.txt.",
        "title": "cmake error 'the source does not appear to contain CMakeLists.txt'"
    },
    {
        "log": "/bin/bash: line 10: ndpiReader: command not found",
        "title": "haproxy: command not found"
    },
    {
        "log": "/bin/bash: line 9: ndpiReader: command not found",
        "title": "haproxy: command not found"
    },
    {
        "log": "/bin/bash: line 7: ndpiReader: command not found",
        "title": "haproxy: command not found"
    },
    {
        "log": "Err:1 http://archive.ubuntu.com/ubuntu focal InRelease",
        "title": "apt-get update fails to fetch files, \u201cTemporary failure resolving \u2026\u201d error"
    },
    {
        "log": "Err:2 http://security.ubuntu.com/ubuntu focal-security InRelease",
        "title": "apt-get update fails to fetch files, \u201cTemporary failure resolving \u2026\u201d error"
    },
    {
        "log": "Temporary failure resolving 'security.ubuntu.com'",
        "title": "apt-get update fails to fetch files, \u201cTemporary failure resolving \u2026\u201d error"
    },
    {
        "log": "Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused) Could not connect to archive.ubuntu.com:80 (185.125.190.83), connection timed out Could not connect to archive.ubuntu.com:80 (185.125.190.82), connection timed out Could not connect to archive.ubuntu.com:80 (185.125.190.81), connection timed out",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Err:2 http://archive.ubuntu.com/ubuntu focal-updates InRelease",
        "title": "apt-get update fails to fetch files, \u201cTemporary failure resolving \u2026\u201d error"
    },
    {
        "log": "Err:3 http://archive.ubuntu.com/ubuntu focal-backports InRelease",
        "title": "apt-get update fails to fetch files, \u201cTemporary failure resolving \u2026\u201d error"
    },
    {
        "log": "Err:4 http://security.ubuntu.com/ubuntu focal-security InRelease",
        "title": "apt-get update fails to fetch files, \u201cTemporary failure resolving \u2026\u201d error"
    },
    {
        "log": "W: Some index files failed to download. They have been ignored, or old ones used instead.",
        "title": "E: Some index files failed to download. They have been ignored, or old ones used instead after apt-get update"
    },
    {
        "log": "Err:1 http://security.ubuntu.com/ubuntu focal-security InRelease",
        "title": "apt-get update fails to fetch files, \u201cTemporary failure resolving \u2026\u201d error"
    },
    {
        "log": "Err:2 http://archive.ubuntu.com/ubuntu focal InRelease",
        "title": "apt-get update fails to fetch files, \u201cTemporary failure resolving \u2026\u201d error"
    },
    {
        "log": "Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.39), connection timed out Could not connect to archive.ubuntu.com:80 (185.125.190.81), connection timed out Could not connect to archive.ubuntu.com:80 (185.125.190.82), connection timed out Could not connect to archive.ubuntu.com:80 (185.125.190.36), connection timed out [IP: 185.125.190.39 80]",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) [IP: 185.125.190.39 80]",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused) Could not connect to archive.ubuntu.com:80 (185.125.190.83), connection timed out Could not connect to archive.ubuntu.com:80 (185.125.190.82), connection timed out Could not connect to archive.ubuntu.com:80 (185.125.190.81), connection timed out",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Could not connect to archive.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/focal/InRelease Could not connect to archive.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/focal-updates/InRelease Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/focal-backports/InRelease Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "W: Failed to fetch http://security.ubuntu.com/ubuntu/dists/focal-security/InRelease Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/focal/InRelease Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/focal-updates/InRelease Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/focal-backports/InRelease Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "W: Failed to fetch http://security.ubuntu.com/ubuntu/dists/focal-security/InRelease Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Err:3 http://archive.ubuntu.com/ubuntu focal-updates InRelease",
        "title": "apt-get update fails to fetch files, \u201cTemporary failure resolving \u2026\u201d error"
    },
    {
        "log": "Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Err:4 http://archive.ubuntu.com/ubuntu focal-backports InRelease",
        "title": "apt-get update fails to fetch files, \u201cTemporary failure resolving \u2026\u201d error"
    },
    {
        "log": "W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/focal/InRelease Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/focal-updates/InRelease Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/focal-backports/InRelease Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "W: Failed to fetch http://security.ubuntu.com/ubuntu/dists/focal-security/InRelease Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused) Could not connect to archive.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/focal/InRelease Temporary failure resolving 'archive.ubuntu.com'",
        "title": "apt-get update fails to fetch files, \u201cTemporary failure resolving \u2026\u201d error"
    },
    {
        "log": "W: Failed to fetch http://security.ubuntu.com/ubuntu/dists/focal-security/InRelease Temporary failure resolving 'security.ubuntu.com'",
        "title": "apt-get update fails to fetch files, \u201cTemporary failure resolving \u2026\u201d error"
    },
    {
        "log": "Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.39), connection timed out Could not connect to archive.ubuntu.com:80 (185.125.190.81), connection timed out Could not connect to archive.ubuntu.com:80 (185.125.190.82), connection timed out Could not connect to archive.ubuntu.com:80 (185.125.190.36), connection timed out [IP: 185.125.190.39 80]",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) [IP: 185.125.190.39 80]",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused) Could not connect to archive.ubuntu.com:80 (185.125.190.83), connection timed out Could not connect to archive.ubuntu.com:80 (185.125.190.82), connection timed out Could not connect to archive.ubuntu.com:80 (185.125.190.81), connection timed out",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Could not connect to archive.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/focal/InRelease Could not connect to archive.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/focal-updates/InRelease Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/focal-backports/InRelease Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "W: Failed to fetch http://security.ubuntu.com/ubuntu/dists/focal-security/InRelease Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/focal/InRelease Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/focal-updates/InRelease Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/focal-backports/InRelease Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "W: Failed to fetch http://security.ubuntu.com/ubuntu/dists/focal-security/InRelease Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/focal/InRelease Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/focal-updates/InRelease Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/focal-backports/InRelease Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "W: Failed to fetch http://security.ubuntu.com/ubuntu/dists/focal-security/InRelease Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused) Could not connect to archive.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/focal/InRelease Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused) Could not connect to archive.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/focal-updates/InRelease Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/focal-backports/InRelease Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "W: Failed to fetch http://security.ubuntu.com/ubuntu/dists/focal-security/InRelease Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "/bin/bash: line 8: ndpiReader: command not found",
        "title": "haproxy: command not found"
    },
    {
        "log": "/bin/bash: line 11: ndpiReader: command not found",
        "title": "haproxy: command not found"
    },
    {
        "log": "ERROR: '~ntop' user or team does not exist.",
        "title": "Cannot add PPA - \"user or team does not exist\""
    },
    {
        "log": "Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.39), connection timed out Could not connect to archive.ubuntu.com:80 (185.125.190.81), connection timed out Could not connect to archive.ubuntu.com:80 (185.125.190.82), connection timed out Could not connect to archive.ubuntu.com:80 (185.125.190.36), connection timed out [IP: 185.125.190.39 80]",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) [IP: 185.125.190.39 80]",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused) Could not connect to archive.ubuntu.com:80 (185.125.190.83), connection timed out Could not connect to archive.ubuntu.com:80 (185.125.190.82), connection timed out Could not connect to archive.ubuntu.com:80 (185.125.190.81), connection timed out",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Could not connect to archive.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/focal/InRelease Could not connect to archive.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/focal-updates/InRelease Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/focal-backports/InRelease Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "W: Failed to fetch http://security.ubuntu.com/ubuntu/dists/focal-security/InRelease Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/focal/InRelease Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/focal-updates/InRelease Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/focal-backports/InRelease Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "W: Failed to fetch http://security.ubuntu.com/ubuntu/dists/focal-security/InRelease Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/focal/InRelease Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/focal-updates/InRelease Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/focal-backports/InRelease Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "W: Failed to fetch http://security.ubuntu.com/ubuntu/dists/focal-security/InRelease Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused) Could not connect to archive.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/focal/InRelease Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused) Could not connect to archive.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/focal-updates/InRelease Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/focal-backports/InRelease Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "W: Failed to fetch http://security.ubuntu.com/ubuntu/dists/focal-security/InRelease Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "/bin/bash: line 9: ntopng: command not found",
        "title": "haproxy: command not found"
    },
    {
        "log": "Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.39), connection timed out Could not connect to archive.ubuntu.com:80 (185.125.190.81), connection timed out Could not connect to archive.ubuntu.com:80 (185.125.190.82), connection timed out Could not connect to archive.ubuntu.com:80 (185.125.190.36), connection timed out [IP: 185.125.190.39 80]",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) [IP: 185.125.190.39 80]",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused) Could not connect to archive.ubuntu.com:80 (185.125.190.83), connection timed out Could not connect to archive.ubuntu.com:80 (185.125.190.82), connection timed out Could not connect to archive.ubuntu.com:80 (185.125.190.81), connection timed out",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Could not connect to archive.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/focal/InRelease Could not connect to archive.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/focal-updates/InRelease Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/focal-backports/InRelease Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "W: Failed to fetch http://security.ubuntu.com/ubuntu/dists/focal-security/InRelease Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/focal/InRelease Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/focal-updates/InRelease Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/focal-backports/InRelease Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "W: Failed to fetch http://security.ubuntu.com/ubuntu/dists/focal-security/InRelease Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/focal/InRelease Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/focal-updates/InRelease Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/focal-backports/InRelease Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "W: Failed to fetch http://security.ubuntu.com/ubuntu/dists/focal-security/InRelease Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused) Could not connect to archive.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/focal/InRelease Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused) Could not connect to archive.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/focal-updates/InRelease Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/focal-backports/InRelease Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "W: Failed to fetch http://security.ubuntu.com/ubuntu/dists/focal-security/InRelease Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::101). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.82). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::103). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.81). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::102). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.36). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::19). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.39). - connect (111: Connection refused) Cannot initiate the connection to security.ubuntu.com:80 (2620:2d:4000:1::16). - connect (101: Network is unreachable) Could not connect to security.ubuntu.com:80 (185.125.190.83). - connect (111: Connection refused)",
        "title": "\"Cannot initiate the connection to in.archive.ubuntu.com:80\" when trying to run \"sudo apt-get update\" behind proxy"
    },
    {
        "log": "fatal: not a git repository (or any of the parent directories): .git",
        "title": "fatal: Not a git repository (or any of the parent directories): .git [duplicate]"
    },
    {
        "log": "Proxy 'backend-app' has no servers available!",
        "title": "Backend app has no server available!"
    },
    {
        "log": "add-apt-repository: command not found",
        "title": "Error sudo: add-apt-repository: command not found"
    },
    {
        "log": "[ERRCODE: SC_ERR_NO_RULES(42)] - No rule files found in /etc/suricata/rules",
        "title": "<Warning> - [ERRCODE: SC_ERR_NO_RULES(42)] - No rule files match the pattern /var/lib/suricata/rules/local.rules"
    },
    {
        "log": "ntopng: Could not connect to Redis at 127.0.0.1:6379 \u2014 Connection refused",
        "title": "Could not connect to Redis at 127.0.0.1:6379: Connection refused with homebrew"
    },
    {
        "log": "iptables-persistent not found. Attempting to install via apt...",
        "title": "Failed to start iptables-persistent.service"
    },
    {
        "log": "iptables-restore: line 3 failed",
        "title": "failed to apply firewall rules with iptables-restore"
    },
    {
        "log": "iptables v1.8.7: unknown option \"--dport\"",
        "title": "iptables error: unknown option --dport"
    },
    {
        "log": "iptables: No chain/target/match by that name.",
        "title": "How to fix \u201ciptables: No chain/target/match by that name\u201d?"
    },
    {
        "log": "iptables: Resource temporarily unavailable",
        "title": "What can cause a \u201cResource temporarily unavailable\u201d on sock send() command"
    },
    {
        "log": "FailedScheduling: 0/5 nodes are available: insufficient memory.",
        "title": "Pod don't run, insufficient resources"
    },
    {
        "log": "Readiness probe failed: HTTP probe failed with statuscode: 503",
        "title": "Getting Readiness & Liveliness probe failed: HTTP probe failed with statuscode: 503 during starting of Varnish using Helm Chart Kubernetes\""
    },
    {
        "log": "rpc error: code = Unknown desc = failed to create Pod sandbox",
        "title": "Failed create pod sandbox: rpc error: code = Unknown desc = NetworkPlugin cni failed to set up pod network [closed]"
    }
]